Cargo.lock

```diff
@@ -4021,9 +4022,8 @@ checksum = "957e51f3646910546462e67d5f7599b9e4fb8acdd304b087a6494730f9eebf04"
 
 [[package]]
 name = "unique_port"
-version = "0.1.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "d79d1fad39d9571050e827f7a33fde5a50952f50e3ccbc007a6566881b17df5d"
+version = "0.2.1"
+source = "git+https://github.com/ales-tsurko/iroha2-unique_port?rev=09e95bb3608626c995b8e7e51c64ba7486c88948#09e95bb3608626c995b8e7e51c64ba7486c88948"
```

Comment:
Do you think we need to manually maintain this? I [nearly spoiled it](https://github.com/hyperledger/iroha/pull/2394#discussion_r906047312)

Reply:
I think, if not automatically, but with `cargo update`, it will change the link to crates.io. Because git in manifest is supposed to be fallback in case if the specified version is not available yet.